libs/ui/src/admin/NewsCardBase/types.ts

```diff
@@ -0,0 +1,22 @@
+import { ReactElement } from 'react'
+
+import { ButtonProps } from '@chakra-ui/react'
+import { News } from '@wsvvrijheid/types'
+
+export type NewsCardBaseProps = {
+  hideDescription?: boolean
+  news: News
+  variant?: 'horizontal' | 'vertical'
+  isBookmarked: boolean
+  onBookmark: (id: string) => void
+  onShare: (id: string) => void
+  onRecommend: () => void
+  onView: (url: string) => void
+}
```

Comment:
The reason I'm using `RecommendedTopic` instead of `RecommendedNews` is that I couldn't use `News` as a database table name because it was complaining about pluralization the names.

Maybe it's a better idea to use `Topics`, `TopicCardBase` and `TopicCard` to avoid the confusion.

```suggestion
import { Topic } from '@wsvvrijheid/types'

export type TopicCardBaseProps = {
  hideDescription?: boolean
  topic: Topic
  variant?: 'horizontal' | 'vertical'
  isBookmarked: boolean
  onBookmark: (id: string) => void
  onShare: (id: string) => void
  onRecommend: () => void
  onView: (url: string) => void
}
```